### PR TITLE
Pin versions for CI success

### PR DESCRIPTION
**Pull request by fastjames**
This extension seems to have problems with rails 5.2.2 and Solidus 2.8.0
alpha (aka master). By pinning these versions we should be able to get
green specs and add CI coverage.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          name: install dependencies
+          command: |
+            bundle install --jobs=4 --retry=3 --path vendor/bundle
+
+      - save_cache:
+          paths:
+            - ./vendor/bundle
+          key: v1-dependencies-{{ checksum "Gemfile.lock" }}
+
+      # - run:
```
Comment (LukeSettle) [inline .circleci/config.yml]: Are you planning on eventually doing this matrix stuff?

Comment (LukeSettle) [reply .circleci/config.yml]: Think we can remove this if not?

Comment (fastjames) [reply .circleci/config.yml]: Removed.

**Review by LukeSettle — APPROVED** [review]
🙌

**Pull request merged**